privacy-protections/storage-blocking/iframe.js

```diff
@@ -31,33 +31,28 @@ function storeData (randomNumber) {
 }
 
 function retrieveData () {
-    return Promise.all(commonTests.map(test => {
+    return Promise.all(commonTests.map(async (test) => {
         try {
             const result = test.retrive();
+            const value = (result instanceof Promise) ? await result : result;
```

Comment:
Await also works on non-promises.
```suggestion
            const value = await result;
```

Reply:
TIL, this is great. Thanks!